configure

```diff
@@ -903,7 +903,7 @@ cat > conftest.$ac_ext << EOF
 #line 904 "configure"
 #include "confdefs.h"
 
-main(){return(0);}
+int main(){return(0);}
```

Comment:
Changes to configure are lost the next time autoconf is used. You need to change configure.in and aclocal.m4 instead.

Reply:
Apparently this is caused by changes with Xcode / clang 12. See e.g. https://sourceforge.net/p/schilytools/tickets/25/?limit=25.

I will redraw this part from this PR. I will take another good look on this problem

Reply:
Dropped from this PR.